helm_deploy/hmpps-book-secure-move-frontend/templates/_helpers.tpl

```diff
@@ -0,0 +1,61 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
This helper looks generic - is it possible in the future for it to live somewhere else instead, to enable reuse by our other apps?

Reply:
It is very generic yes and duplicated all over, however there isn't an easy way to include such functions with helm.  One alternative  is to have to publish a generic chart for all application to use - but that has drawbacks - experimental repo here for such an idea:  https://github.com/ministryofjustice/hmpps-helm-charts